src/components/auth/LoginForm.jsx

```
@@ -0,0 +1,75 @@
+        autoComplete="current-password"
+        value={password}
+        onChange={(e) => setPassword(e.target.value)}
+      />
+
+      <Button type="submit" fullWidth variant="contained" sx={{ mt: 3, mb: 2 }} disabled={loading}>
+        {loading ? <CircularProgress size={24} /> : 'Sign In'}
+      </Button>
+
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Link component="button" variant="body2" onClick={() => onToggleForm('register')}>
+          Don't have an account? Sign Up
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)